recipes/log4cxx/all/conanfile.py

```diff
@@ -62,20 +64,19 @@ def _compilers_minimum_version(self):
             "apple-clang": "10",
         }
 
-    def _validate_compiler_settings(self):
-        compiler = self.settings.compiler
-        if compiler.get_safe("cppstd"):
+    def validate(self):
+        # TODO: if compiler doesn't support C++17, boost can be used instead
```

Comment:
I remember I was using log4cxx long before even C++14 or C++11, even on some embedded devices. does it really require C++17? 👀

Reply:
Good question, same feeling.

Reply:
It's been a year since it was changed https://github.com/apache/logging-log4cxx/blame/e1b363fc9405ac0728cfeac104a3aa729de1c7bd/CMakeLists.txt#L59

Reply:
Well, I used log4cxx 10 years ago :stuck_out_tongue_closed_eyes: